### PR TITLE
Update exercise 1

### PR DESCRIPTION
**Pull request by maryx**
Updates exercise 1 to be better documented and cleaned up somewhat.

### CODE REVIEW

File: course/01_hello_rectangle/solution_01_hello_rectangle/README.md [hunk 1]
```diff
@@ -1,14 +1,22 @@
-# solution_01_hello_rectangle
+# Build a Hello Rectangle App
 
-This is the solution to the Hello Rectangle app.
+## Goals
+ - Verify you've set up Android Studio
+ - Verify that the Flutter plugin has been installed
+ - Run an app!
```
Comment (yjbanov) [inline course/01_hello_rectangle/solution_01_hello_rectangle/README.md]: nit: some points in the lists end with punctuation, others don't. Perhaps adopt one style for consistency (guessing _with_ punctuation is the way to go, as some points have multiple sentences).

**Comment by yjbanov** [comment]
lgtm

File: course/01_hello_rectangle/solution_01_hello_rectangle/README.md (hunk 1 above)
Comment (maryx) [reply course/01_hello_rectangle/solution_01_hello_rectangle/README.md]: Good point, I'll use punctuation. I'll make this change for all remaining exercises too.

**Comment by PatrickPrakash** [comment]
This code is not working on my flutter beta 3

**Comment by PatrickPrakash** [comment]
Please update the code

**Comment by maryx** [comment]
@PatrickPrakash Please see https://github.com/flutter/udacity-course/issues/112. 
What version of Flutter and Android Studio are you using?
new and const are not required with Flutter Beta 3.x (such as from here), and Android Studio 3.1.x, and the Flutter v24.0 plugin in Android Studio.

**Pull request merged**